develpoedCalculator.js

```diff
@@ -1 +1,8 @@
-class DevelopedCalculator extends Calculator {};
\ No newline at end of file
+class DevelopedCalculator extends Calculator {
+    constructor(selector) {
+        this.hostElem1 = document.querySelector(selector);
+    }
+
+}
+let developedCalculator = new DevelopedCalculator("#hostElem1");
```

Comment:
can you please point me where this selector  ("#hostElem1") is used in HTML code?

Reply:
sorry it was one in the html file 
but I remembered maybe you do not prefer using numbers 
so I changed it to _
but I forgot to change it in html file also